packages/bridge-ui-v2/src/libs/chain/chains.ts

```diff
@@ -21,7 +21,7 @@ import {
 } from '$env/static/public';
 
 export const mainnetChain: Chain = {
-  id: parseInt(PUBLIC_L1_CHAIN_ID),
+  id: +PUBLIC_L1_CHAIN_ID,
```

Comment:
Everytime I see this, it makes me love and hate Javascript at the same time :D

Reply:
If you prefer I can remove this magic. Lucky you I haven't yet use double tilde `~~` 😉

I was parsing the int, but since we get this from an env var as a string, and `Chain` from `@wagmi/core` type requires the chain.id to be int, I decided to make it quicker by converting using this trick